modules/ndu/src/backend/conditions.ts

```diff
@@ -73,7 +73,7 @@ export const dialogConditions: sdk.Condition[] = [
     label: 'Raw JS expression',
     description: `{label}`,
     params: {
-      expression: { label: 'Expression to evaluate', type: 'string' },
+      expression: { label: 'Expression to evaluate', type: 'string', rows: 5 },
```

Comment:
what's the rows param ? is this for the ui ?

Reply:
Yes. The UI & everything associated with a condition is defined here. It makes it easier to make changes or to create new ones. The logic will also be used for prompts or packaged skills